### PR TITLE
The great babelification

### PR DESCRIPTION
**Pull request by nevir**

### CODE REVIEW

File: scripts/devserver [hunk 1]
```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env ./node_modules/.bin/babel-node
```
Comment (aowainati) [inline scripts/devserver:1]: How much do you care about Javascript scripts having a `.js` file extension? Makes git diffs nice and colorful, makes it easy to see at a glance what language the scripts are in (we could also rename the bash ones to `.sh`). Thoughts?

File: scripts/devserver [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+import * as path from 'path';
+import devMiddleware from 'webpack-dev-middleware';
+import express from 'express';
+import hostMiddleware from 'webpack-hot-middleware';
```
Comment (aowainati) [inline scripts/devserver]: Typo, `hotMiddleware`

File: scripts/devserver (hunk 1 above)
Comment (nevir) [inline scripts/devserver:1]: I dunno about your editor, but mine...

File: scripts/devserver (hunk 2 above)
Comment (nevir) [inline scripts/devserver]: FECK

File: scripts/devserver (hunk 1 above)
Comment (aowainati) [inline scripts/devserver:1]: DON'T MAKE ME SLAP YOU

Also by git diff I mean pull request diff!

**Comment by aowainati** [comment]
image

**Pull request merged**